element_calcium_imaging/scan.py

```diff
@@ -136,11 +156,50 @@ class Scan(dj.Manual):
     -> Session
     scan_id: int        
     ---
-    -> Equipment  
+    -> [nullable] Equipment  
     -> AcquisitionSoftware  
     scan_notes='' : varchar(4095)         # free-notes
     """
 
+    @classmethod
+    def auto_generate_scan(cls, session_key):
```

Comment:
This doesn't quite work the way we would intend it to. It's nice to keep it here for other people to "use at your own risk" but now I don't think it's wise.
I'd suggest removing this function all together, and also remove `get_session_directory` requirement as well.

Reply:
@kabilar is suggesting that we use only `get_session_directory` instead of `get_XXX_files` functions because TIF images are not always associated with the ScanImage files. Regardless of this discussion, I'll remove the `auto_generate_scan` method.